Coin config additions

Gumbees opened the pull request:
Added vot.json and zel.json to the coins and pool_configs. Please wait until Pull Request #1 is merged. There are some out of date config files compared to that branch.

Gumbees committed:
added zel.json

Gumbees committed:
added votecoin

Gumbees committed:
added vot.json pool_config

Gumbees committed:
zel didn't push to this branch under pool_configs.

egyptianbman commented on pool_configs/vot.json:
```diff
@@ -0,0 +1,87 @@
+    },
+
+    "paymentProcessing": {
+	"minConf": 10,
```
Can  you fix the alignment on this line?

Gumbees replied:
done.

egyptianbman commented on pool_configs/vot.json:
```diff
@@ -0,0 +1,87 @@
+
+    "tlsOptions": {
+        "enabled": false,
+        "serverKey":"",
```
I also fixed lack of space after the colon on several of these in the other PR.

Gumbees replied:
sounds good. i went through and cleaned up the way it looked anyways and a few other things like the correct port numbers. let me know if vot is correct.

Gumbees committed:
Corrected port and spacing

Spacing should be corrected. let me know if its still off. Looks good in notepad++.

Added correct RPC port for zel.json.

Gumbees committed:
spacing fix

Gumbees committed:
Corrected default rpc port

Found a commit on votecoin GitHub for a change to 8242

Gumbees reviewed (commented):
I need to update the default rpcuser and rpcpass.

Gumbees commented:
v2 has zel and vot in it. deleting.
